plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastoreModule.java

```diff
@@ -45,7 +45,9 @@ protected void setup(Binder binder)
             bindMetastoreModule("thrift", new ThriftMetastoreModule());
             bindMetastoreModule("file", new FileMetastoreModule());
             bindMetastoreModule("glue", new GlueMetastoreModule());
-            bindMetastoreModule("alluxio", new AlluxioMetastoreModule());
+            // Alluxio is not maintained and accumulates security vulnerabilities,
```

Comment:
`Alluxio is not maintained by Trino community and accumulates security vulnerabilities`?

Reply:
Done, thanks